fontforge/featurefile.c

```diff
@@ -456,20 +456,21 @@ static OTLookup *lookup_in_rule(struct fpst_rule *r,int seq,int *index, int *pos
     /* That's legal in otf, but I don't think it can be specified in the */
     /*  feature file. It doesn't seem likely to be used so I ignore it */
 
-    for ( i=0; i<r->lookup_cnt && seq<r->lookups[i].seq; ++i );
-    if ( i>=r->lookup_cnt )
-return( NULL );
+    for ( i=0; i<r->lookup_cnt && seq>r->lookups[i].seq; ++i );
+    if ( i > 0 && seq != r->lookups[i].seq )
```

Comment:
If `i` equals `r->lookup_cnt` here, `r->lookups[i].seq` is referring to garbage off the end of the list.

Reply:
Fixed!